node/builder.go

```diff
@@ -369,6 +369,7 @@ func ConfigBoost(c interface{}) Option {
 	if !ok {
 		return Error(xerrors.Errorf("invalid config from repo, got: %T", c))
 	}
+	cfg.Storage.ParallelFetchLimit = 10
```

Comment:
@aarshkshah1992 maybe add this to the itest, rather than here, if this is used only for tests.

Reply:
You mean before instantiating the itest ?

Reply:
while instantiating Boost in tests, you can set any config you need. since this config is necessary for tests, and not production code, it can live in the test `framework`.